third_party/amd/lib/TritonAMDGPUToLLVM/BuiltinFuncToLLVM.cpp

```diff
@@ -19,8 +19,9 @@ namespace {
 
 class CallOpConversion : public mlir::RewritePattern {
 public:
-  CallOpConversion(mlir::MLIRContext *context)
-      : mlir::RewritePattern(LLVM::CallOp::getOperationName(), 1, context) {}
+  CallOpConversion(mlir::MLIRContext *context, bool ftz)
```

Comment:
While here can you update this pattern to use `OpRewritePattern`? It would be a bit better with that.